### ISSUE TITLE
fix meeting invite link

### ISSUE THREAD

**Issue by ericdill** [conda-forge/conda-forge.github.io#816]
Issue:

<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list

```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info

```
</details>

**Comment by jakirkham** [comment]
Is it 16:30 GMT? I thought it was 1p EDT so 17:00 GMT.

**Comment by ericdill** [comment]
yeah you're 100% right. I did not successfully read my calendar 😞

**Comment by jakirkham** [comment]
No worries.

We might as well determine how we want to handle DST (if at all). I think we have used US based DST. Though we don’t necessarily need to do that if something else makes sense.

**Comment by ericdill** [comment]
Pinning it to GMT0 I think makes the most sense. Then we all just adjust in whatever time zone we are in

**Comment by jakirkham** [comment]
Is it safe to close this issue Eric? Or is there something still actionable here?

**Comment by carterbox** [comment]
I'm here because some of the meeting links are still broken/incorrect. In my observation:

- Link to Zoom on meeting minutes page of rendered conda-forge is correct.
- Link to Zoom inside meeting minutes on hackmd is not correct.
- Link to Google Calendar in conda-forge.github.io/README is incorrect; the repeat expired in 2019, so it no longer adds future meetings. Also, I think the Zoom meeting inside that invite is also incorrect.
- README says every two weeks, but arewemeetingyet calendar says the meeting is every week.

**Comment by jakirkham** [comment]
First off apologies that this didn't get communicated well.

We actually canceled this week (and the remainder of the meetings this year). Next meeting will be on the 12th of January.

FWIW the time would have been right had we met. So should be correct for the next scheduled meeting.
